components/ActivitiesList/index.js

```diff
@@ -1,24 +1,35 @@
-import ActivityCardPreview from "./ActivityCardPreview";
-import activities from "../../assets/activities";
+import ActivityCardPreview from "../ActivityCardPreview";
 import styled from "styled-components";
 
 export const StyledUl = styled.ul`
   list-style: none;
   padding: 0;
 `;
 
-const StyledListItem = styled.li`
+export const StyledListItem = styled.li`
```

Comment:
Do you want to use this styled component anywhere else? To define global styled Components you should use a separate (global) file.